WalletWasabi.Gui/Tabs/WalletManager/GenerateWalletViewModel.cs

```diff
@@ -87,6 +94,24 @@ private bool ValidateWalletName(string walletName)
 			return isValid && !isReserved;
 		}
 
+		public string ValidatePassword()
+		{
+			string password = Password;
+			List<string> messages = new List<string>();
+			if (PasswordHelper.IsTrimable(password, out _))
+			{
+				messages.Add("Leading and trailing white spaces are not enabled!");
```

Comment:
```suggestion
				messages.Add("Leading and trailing white spaces are not allowed!");
```
Saying `are not enabled` hints that they can be enabled somehow.